.github/workflows/ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
         matrix:
-          node-version: ['lts/gallium']
+          node-version: ['lts/hydrogen']
```

Comment:
On the smaller `npm` package components we had this also using node 20. Here I opted to not do that because It added significantly more steps to the CI process since these are broken up into shards. Not sure if we do want to do that here as well

Reply:
From my forked workflow with both node 20 and lts/hydrogen in the jest test workflow

![image](https://github.com/nasa/earthdata-search/assets/34591886/eab61074-9c17-4ed7-8100-8c15b64b7c5c)

Reply:
We don't want to do that here, this isn't a reusable library where people might have different node versions, it is an application with a specific node requirement